AsaApiLoader.sln

```diff
@@ -21,6 +23,14 @@ Global
 		{9DBA82CD-8DAF-47BC-A132-483F313B25F6}.Release|x64.Build.0 = Release|x64
 		{9DBA82CD-8DAF-47BC-A132-483F313B25F6}.Release|x86.ActiveCfg = Release|Win32
 		{9DBA82CD-8DAF-47BC-A132-483F313B25F6}.Release|x86.Build.0 = Release|Win32
+		{39E23980-84DF-4C64-BDBD-C55BE240FD0E}.Debug|x64.ActiveCfg = Debug|x64
+		{39E23980-84DF-4C64-BDBD-C55BE240FD0E}.Debug|x64.Build.0 = Debug|x64
+		{39E23980-84DF-4C64-BDBD-C55BE240FD0E}.Debug|x86.ActiveCfg = Debug|Win32
```

Comment:
I'm going to approve the PR, but we should remove the 32 bit build options as we need to keep our library as the same architecture as the server.